app/components/subcases/subcase-header.js

```diff
@@ -75,6 +77,18 @@ export default class SubcasesSubcaseHeaderComponent extends Component {
           this.meetingIsClosed = true;
         }
       }
+      const submissions = yield this.args.subcase.submissions;
+      let draftPieceWithoutAccepted = false;
+      for (const submission of submissions) {
+        const pieces = yield submission.pieces;
+        for (const piece of pieces) {
+          const actualPiece = yield piece.acceptedPiece;
+          if (!actualPiece) {
+            draftPieceWithoutAccepted = true;
+          }
+        }
+      }
+      this.canSubmitNewDocuments = !draftPieceWithoutAccepted;
```

Comment:
One small remark here: since the `currentSubmission` is retrieved using `getOngoingSubmissionForSubcase`, in the case where `this.canSubmitNewDocuments` is `false` because the documents have not propagated yet, the button `Huidige indiening` will not be shown, as it has already been treated. 

So there is a small window of time where the kabinetsdossierbeheerder cannot see the status of their submitted files.